Show hint that you cannot manage content with Katello when using Foreman on Debian (#2400)
theforeman/foreman-documentation: guides/common/modules/snip_no-content-management-for-foreman-on-deb.adoc

> Foreman on Debian/Ubuntu does to support the Katello plug-in required to manage content.

Reviewer: Isn't it the other way around?
```suggestion
The Katello plug-in to Foreman is not supported on Debian/Ubuntu.
```

Author: Slightly reworded and applied.
